app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -132,6 +134,8 @@ class DefaultToolbarMenu(
             context.components.browsingModeManager.mode == BrowsingMode.Normal
         val shouldDeleteDataOnQuit = Settings.getInstance(context)
             .shouldDeleteBrowsingDataOnQuit
+        val shouldShowWebcompatReporter =
+            Config.channel != ReleaseChannel.FenixProduction
```

Comment:
You want to add `FennecProduction` here too.

Reply:
Would it be a good idea for me to add an `isRelease` check into `app/src/main/java/org/mozilla/fenix/Config.kt` in addition to `isReleaseOrBeta`, or is this use-case too niche? :)